media-gfx/fontforge/fontforge-20190501~git.recipe

```diff
@@ -112,9 +112,7 @@ BUILD()
 	export LDFLAGS="-lnetwork $LDFLAGS"
 	./bootstrap
 	runConfigure --omit-dirs binDir ./configure \
-		--prefix=$commandBinDir \
 		--enable-python-scripting=3
-		false
```

Comment:
That was still in the phase :) you also removed $commandBinDir?

Reply:
Yeah, it would install in $prefix/bin instead of $prefix

Reply:
That was the intention https://github.com/haikuports/haikuports/blob/7d74d6d048fe65a06c1fe8b6709da832bc031899/media-gfx/fontforge/fontforge-20190501%7Egit.recipe#L23

Reply:
ah jikes! should be --bindir=$commandBinDir :)

Reply:
I added --bindir now